test/e2e/persistent_volumes.go

```diff
@@ -66,58 +66,96 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 			persistentVolumeTestCleanup(c, config)
 		}()
 
+		// Create the nfs server pod
 		pod := startVolumeServer(c, config)
 		serverIP := pod.Status.PodIP
 		framework.Logf("NFS server IP address: %v", serverIP)
 
+		// Define the PersistentVolume and PersistentVolumeClaim
 		pv := makePersistentVolume(serverIP)
 		pvc := makePersistentVolumeClaim(ns)
 
-		framework.Logf("Creating PersistentVolume using NFS")
+		// Create the PersistentVolume and wait for PersistentVolume.Status.Phase to be Available
+		framework.Logf("Creating PersistentVolume")
 		pv, err := c.PersistentVolumes().Create(pv)
-		Expect(err).NotTo(HaveOccurred())
+		if err != nil {
+			framework.Failf("PersistentVolume errored during creation: %v", err)
+		}
+		framework.WaitForPersistentVolumePhase(api.VolumeAvailable, c, pv.Name, 1*time.Second, 20*time.Second)
```

Comment:
On my test laptop it takes *considerably* longer than 20s for the claim to bind to the nfs pv. I see no downside to waiting 5m or so to ensure that the binding has sufficient time. In the working case it still binds quickly. In the non-working case you have to wait 5m to see the failure. But better to wait than have the test fail because it gave up early.